flink-contrib/flink-storm-examples/src/main/java/org/apache/flink/storm/exclamation/ExclamationWithBolt.java

```diff
@@ -72,7 +71,7 @@ public static void main(final String[] args) throws Exception {
 				.transform("StormBoltTokenizer",
 						TypeExtractor.getForObject(""),
 						new BoltWrapper<String, String>(new ExclamationBolt(),
-								new String[] { Utils.DEFAULT_STREAM_ID }))
+								"stream", "component", new String[] { Utils.DEFAULT_STREAM_ID }))
```

Comment:
Can't we use default values for streamID and componentID here, ie, obit both parameters?